help/include/plan-upgrade-steps.md

```diff
@@ -2,4 +2,9 @@
 
 1. Click **Add card** to enter your payment details.
 
+1. *(optional)* **View and update** billing information included on
+   receipts so that they are different from the information entered for
+   the payment method, e.g. in case you would prefer that the company's
+   name be on receipts instead of the card holder's name.
```

Comment:
I updated this so that the text here matches what is on the upgrade billing page, e.g. "View and updated billing information".